lib/compute.rb

```diff
@@ -0,0 +1,29 @@
+module Declarations
+  module prefills
```

Comment:
The module name `prefills` should be capitalized to follow Ruby naming conventions for constants and modules. Consider renaming it to `Prefills`.